packages/fxa-settings/src/components/Ready/index.test.tsx

```diff
@@ -27,7 +26,8 @@ describe('Ready', () => {
       'You’re now ready to use Account Settings'
```

Comment:
We might want to align on if we're going with "Account Settings" or "account settings", @vpomerleau's last PR used lowercase.

Reply:
Definitely a great idea. Yeah I'm just recreating what we have